lib/src/main/kotlin/org/didcommx/didcomm/DIDComm.kt

```diff
@@ -174,4 +192,19 @@ class DIDComm(private val didDocResolver: DIDDocResolver, private val secretReso
 
         return unpack(params, recipientKeySelector)
     }
+
+    /**
+     *  Unpacks the packed DIDComm Forward message by doing decryption and verifying the signatures.
+     *  TODO more details
+     *
+     *  @param params Unpack Parameters.
+     *  @return Result of Unpack Forward Operation.
+     */
+    fun unpackForward(params: UnpackParams): UnpackForwardResult {
```

Comment:
Do we really need `unpackForward` in the main `DIDComm` class? I think it should be in another class within `protocol.forward` package (as in Python)